pontoon/base/apps.py

```diff
@@ -16,6 +16,9 @@ def ready(self):
         super(BaseConfig, self).ready()
         self.monkeypatch()
 
+        # Load celery app so shared_task uses it for executing tasks.
```

Comment:
http://celery.readthedocs.org/en/latest/django/first-steps-with-django.html#using-the-shared-task-decorator